bokeh/core/_templates/js_resources.html

```diff
@@ -9,12 +9,45 @@
 :type js_raw: list[str]
 
 #}
-{%- for file in js_files %}
-<script type="text/javascript" src="{{ file }}"></script>
-{%- endfor %}
+<script type="text/javascript">
+    function loadJsFilesSequentially(scriptsCollection, startIndex, librariesLoadedCallback) {
+     if (scriptsCollection[startIndex]) {
+       var fileref = document.createElement('script');
+       fileref.setAttribute("type","text/javascript");
+       fileref.setAttribute("src", scriptsCollection[startIndex]);
```

Comment:
since this is code rather than html I think `fileref.setAttribute("async", false)` _might_ actually work to force these into order, which is how we were doing it in autoload_js.js.
That then avoids "manually" implementing sync loading as you are in the `onload` here.